docker/table-diff-handle/tablediffview/github_api_handle.py

```diff
@@ -0,0 +1,82 @@
+
+import re
+import requests
+from tablediffview.config import CONFIG
+from functools import reduce
+
+GITHUB_API_ISSUE_URL = (
+    f"{CONFIG.GITHUB_API_URL}/{CONFIG.GITHUB_REPO_OWNER}/{CONFIG.GITHUB_REPO_NAME}/issues"
+)
+
+ISSUE_NAME_REGEX = r'main|^release/\d*.\d*|^v\d*.\d*.\d*'
+REQUEST_HEADER = {
+    "Authorization": f"token {CONFIG.GITHUB_TOKEN}",
+    "Content-Type": "application/vnd.github+json"
+}
+@staticmethod
+def get_issue_number(branch_name: str):
+    extract_issue_name = re.findall(ISSUE_NAME_REGEX, branch_name)
+    issue_title = "{0} - Tables different"
+    if not extract_issue_name:
```

Comment:
@TruongQuangSB This seems to be twisted. Shouldn't it be 

```suggestion
    if extract_issue_name:
```

Reply:
@mariusheine u are right